### ISSUE TITLE
I.L.: Remove flipper/feature flag and associated code post-launch

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#13594]
## Description

`income_limits_new` was created as a flipper but never used and needs to be removed.

## Acceptance criteria

- [ ] `income_limits_new` is removed from features.yml

**Comment by wesrowe** [comment]
No flipper no-op. Closing.

**Comment by jilladams** [comment]
`income_limits_new` did exist, has been removed from the code. Is still present in the Flipper UI because the Flipper cleanup job still does not run.

**Comment by jilladams** [comment]
@wesrowe @randimays @chriskim2311 reopening. Per Platform: https://dsva.slack.com/archives/CBU0KDSB1/p1693358310021089?thread_ts=1693325134.342619&cid=CBU0KDSB1

> income_limits_new. That is still in features.yml

**Comment by jilladams** [comment]
@chriskim2311 re: how we get this removed from the UI, thread here: 
https://dsva.slack.com/archives/CBU0KDSB1/p1693506345591819?thread_ts=1693325134.342619&cid=CBU0KDSB1

tl;dr: see if you have access to Argo. (Details on tool access here: https://depo-platform-documentation.scrollhelp.site/getting-started/request-access-to-tools).  And if not, open a Platform support request for Backend and ask them to follow the steps in that support thread (until Plat docs are updated) to get it removed from the UI.

**Comment by chriskim2311** [comment]
It doesn't seem like I have write access. I can access the argo tools though. I will open a Platform support request to ask them to remove it.
